app/models/individual_market/application.rb

```diff
@@ -113,6 +113,11 @@ class Application < Sbm::Application
     # @return [Symbol] The current state of the application
     field :current_state, type: Symbol, default: :initial
 
+    # @!attribute is_renewal
+    # @return [Boolean] Indicates if the application is a renewal application
+    # @note This field is used to track if the application is a renewal of a previous application
+    field :is_renewal, type: Boolean, default: false
```

Comment:
- Good one, I wanted this field from very long time, easy to query renewal applications and 
- If possible predecessor_id can also be used for copy applications to know from which application the new application is been copied.